code/modules/mob/living/simple_animal/hostile/necro.dm

```diff
@@ -39,11 +66,12 @@
 
 	environment_smash = 1
 	meat_type = null
-
+/*
 #define EVOLVING 1
-#define MOVING_TO_TARGET 2
+#define MOVING_TO_TARGET 2*/
 #define EATING 3
 #define OPENING_DOOR 4
```

Comment:
Not sure why I've left these uncommented, as busy's now used as a boolean anyway